packages/default/scss/editor/_layout.scss

```diff
@@ -3,6 +3,13 @@
     // Base
     .k-editor {
         @include use-background-clip();
+        border-width: $editor-border-width;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: inherit;
```

Comment:
It doesn't really matter, but for consistency sake we might want to have the `font-family` as a separate variable, as it is in [the other components](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/grid/_layout.scss#L43).